pkg/app/app.go

```diff
@@ -218,6 +219,36 @@ func initConfiguration(engine workflow.Engine, config configuration.Configuratio
 
 	config.AddDefaultValue(configuration.INPUT_DIRECTORY, defaultInputDirectory())
 	config.AddDefaultValue(configuration.PREVIEW_FEATURES_ENABLED, defaultPreviewFeaturesEnabled(engine, logger))
+	config.AddDefaultValue(configuration.CUSTOM_CONFIG_FILES, CustomConfigFiles(config))
+}
+
+func CustomConfigFiles(config configuration.Configuration) configuration.DefaultValueFunction {
```

Comment:
Suggestion: I think there is no need to export this.

```suggestion
func customConfigFiles(config configuration.Configuration) configuration.DefaultValueFunction {
```

Reply:
Let me check - language server had used it in standalone-mode, but maybe it's now obsolete.